python/hail/api2/context.py

```diff
@@ -108,6 +108,11 @@ def import_plink(self, bed, bim, fam, min_partitions=None, delimiter='\\\\s+',
     def read(self, path, drop_samples=False, drop_variants=False):
         return self._hc1.read(path, drop_samples, drop_variants).to_hail2()
 
+    @handle_py4j
+    @typecheck_method(path=oneof(strlike, listof(strlike)))
+    def get_vcf_metadata(self, path):
```

Comment:
should be in methods, not on hail context

Reply:
eventually, but for now I'm treating just like the other import methods while i focus on moving docs for the export methods

Reply:
OK, will be moved over in next round of API assignments